CMakeLists.txt

```diff
@@ -191,6 +191,13 @@ if(${PX4_PLATFORM} STREQUAL "posix")
 	endif()
 endif()
 
+
+option(PX4_EXPORT_RESTRICTION_BUILD "Enable export compliant build of PX4. This adds some flight time and wind restrictions." OFF)
```

Comment:
I'm not sure about carrying this as a cmake option, we won't really have visibility into this actually being in the build or not.

Would it be better in kconfig and then we have a new build px4/fmu-v5x/restricted.px4build (or whatever)? Then it's unambiguous in the build, won't depend on cmake configure time options, we'll be able to verify from `ver all`, and we'll have traceability back to the source.

Reply:
We discussed it and would prefer to keep it as proposed here, as a cmake option. I've added a flag to `ver all` indicating whether it's an export restricted build or not.